models/us_congress_legislators/code/upload.py

```
@@ -0,0 +1,95 @@
+    tb = bd.Table(dataset_id=DATASET_ID, table_id=table_slug)
+    tb.create(
+        path=path,
+        source_format="parquet",
+        if_table_exists="replace",
+        if_storage_data_exists="replace",
+        if_dataset_exists="pass",
+    )
+
+    query = (
+        f"select count(*) as n from "
+        f"`{BILLING_PROJECT}.{DATASET_ID}_staging.{table_slug}`"
```

Ruff (0.15.20)

[error] Possible SQL injection vector through string-based query construction

(S608)